internal/tools/go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	go.opentelemetry.io/build-tools/crosslink v0.23.1
 	go.opentelemetry.io/build-tools/githubgen v0.23.1
 	go.opentelemetry.io/build-tools/multimod v0.23.1
+	go.opentelemetry.io/collector/internal/cmd/pdatagen v0.130.0
```

Comment:
i dont think this will break anything because i don't think anything outside of this repo depends on this directory, but this is an unpublished package version. just making a note in case something breaks somewhere else because of it (and then we need to use the 0.0.0-sha+date version)